migrations/back-37/Hasher.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"strconv"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+
+	"github.com/tidepool-org/platform/data/deduplicator/deduplicator"
+	"github.com/tidepool-org/platform/data/types"
+)
+
+// used for BACK-37 to set hash for jellyfish datum for migrating to platform API
+func CreateHash(bsonData bson.M) string {
+	// represents data from base id fields
+	var baseData = func(bsonData bson.M) []string {
```

Comment:
Any reason this is a function? Why not simply calculate and use the result below.

Reply:
Plus, if you have a value of `baseData` here, then you can simply append to it in the switch and then pull out the `makeHash` below the switch and DRY up the code.